internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/endToEnd/TestStuckConsumerRepair.java

```diff
@@ -79,7 +79,7 @@ public void setUp() {
   private static VeniceClusterWrapper setUpCluster() {
     Properties extraProperties = new Properties();
     extraProperties.setProperty(DEFAULT_MAX_NUMBER_OF_PARTITIONS, "5");
-    VeniceClusterWrapper cluster = ServiceFactory.getVeniceCluster(1, 0, 1, 2, 1000000, false, false, extraProperties);
+    VeniceClusterWrapper cluster = ServiceFactory.getVeniceCluster(1, 0, 1, 1, 1000000, false, false, extraProperties);
```

Comment:
Why reducing server count here? Could you explain a bit? I mean, extra server should not be impacting the test right...?

Reply:
For simplification and to avoid the possibility that one host may not getting any partition assigned, thus not triggering the repair service to kick in at all. If that happens, the following assert would fail.

Reply:
I see, so that is the reason of the flakiness if I understand your comment correctly. Thanks!